spec/features/admin_can_view_items_spec.rb

```diff
@@ -0,0 +1,32 @@
+require 'rails_helper'
+RSpec.describe 'Admin', type: :feature do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.